cmd/cloudshell_open/appfile_test.go

```diff
@@ -88,25 +88,26 @@ func Test_parseAppFile(t *testing.T) {
 		wantErr bool
 	}{
 		{"empty json is EOF", ``, nil, true},
-		{"empty object ok", `{}`, &appFile{}, false},
+		{"empty object ok", `{}`, &appFile{RequireAuth: &fals}, false},
```

Comment:
ditto: using `bool` above solves specifying these values everywhere in the tests.